trackma/ui/qt/settings.py

```diff
@@ -356,10 +356,12 @@ def __init__(self, parent, worker, config, configfile):
 
         # Content
         self.contents = QStackedWidget()
-        self.contents.addWidget(page_media)
-        self.contents.addWidget(page_sync)
-        self.contents.addWidget(page_ui)
-        self.contents.addWidget(page_theme)
+        for page in (page_media, page_sync, page_ui, page_theme):
+            scrollable_page = QScrollArea()
+            scrollable_page.setWidgetResizable(True);
```

Comment:
There's an extraneous semicolon here.

Reply:
The semi-colon muscle memory strikes again!

Thanks for reviewing, I've removed it now.